examples/correlation_alignment_example.py

```diff
@@ -87,20 +85,29 @@
             select_pathway(data.C.mean(indirect), signal_pathway)
         )
         data.setaxis("t2", lambda x: x - best_shift).register_axis({"t2": 0})
+        fl.image(data, ax=ax_list[3])
+        ax_list[3].set_title("Hermitian Test (t)")
         data.ft("t2")
-        data *= mysgn
-        fl.image(data, ax=ax_list[1])
-        ax_list[1].set_title("Phased and Centered")
-        #}}}
-        #{{{ Applying Correlation Routine to Align Data
+        fl.image(data, ax=ax_list[2])
+        ax_list[2].set_title("Hermitian Test (v)")
+        fig.tight_layout(rect=[0, 0.03, 1, 0.95])
+        # }}}
+        # {{{ Applying Correlation Routine to Align Data
+        mysgn = (
+            select_pathway(data, signal_pathway).C.real.sum("t2").run(np.sign)
+        )  # this is the sign of the signal -- note how on the next line,
+        #    I pass sign-flipped data, so that we don't need to worry about
+        #    messing with the original signal
         opt_shift, sigma = correl_align(
-            data*mysgn, indirect_dim=indirect, signal_pathway=signal_pathway, 
-            sigma=50, fl=fl
+            data * mysgn, indirect_dim=indirect, signal_pathway=signal_pathway, sigma=50
```

Comment:
it seems like you had trouble with a merge and didn't merge this in.  This was an important change.